openlcs/products/serializers.py

```diff
@@ -33,19 +33,18 @@ def get_components(self, obj):
             return retval
         # There should be only one root node for each release
         release_node = obj.release_nodes.get()
-        container_nodes = release_node.get_descendants().filter(
-            component__type='OCI'
+        group_nodes = release_node.get_descendants().filter(
+            component__type__in=['OCI', 'RPMMOD']
```

Comment:
Do we need to think add module into container node endpoints?

Reply:
If a module(or any other package types) is a child node of a container, its data should have been collected in the container's descendants.